src/vos/vos_iterator.c

```diff
@@ -328,6 +328,82 @@ vos_iter_finish(daos_handle_t ih)
 	return rc || prc;
 }
 
+static inline void
```

Comment:
Not to reviewers.  Nothing here is different, it just moved here from later in the file